views/layout.erb

```diff
@@ -26,6 +26,12 @@
       <%= erb :navigation %>
       <article class="site-layout__center owl prose">
         <%= styled_flash %>
+
+        <%= erb :message, locals: { message: "Something neutral happened, but hey, pay attention.", kind: '' } %>
```

Comment:
Hey, @niquerio! Would you be able to wire these message varieties with the styled flash logic? Thank you!

Reply:
done!